### ISSUE TITLE
800 series added entry--personal name

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#16]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/8XX.csv

**Comment by lake44me** [comment]
@pennylenger Please note - before this mapping is ready for transform, there may be rows that get [Rreproduction Conditions](https://github.com/uwlib-cams/MARC2RDA/wiki/Reproductions-Guidance) checked and need to have that information added. Let @lake44me and @dchen077 know when the mappings/transform are otherwise reviewed/approved.

**Comment by pennylenger** [comment]
@AdamSchiff A strange question: can 800/810/811 only have name portion, no title portion?

**Comment by lake44me** [comment]
Just look at the LC documentation for 800.  Plenty of name/title examples. https://www.loc.gov/marc/bibliographic/bd800.html

**Comment by AdamSchiff** [comment]
No.  It is a series access field so it must have an access point for a work or expression.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Comment by cspayne** [comment]
@pennylenger, @dchen077 and I ran into a few questions while looking at the 800 mapping. 

1. What should the LCNAF IRI value be when there is no $2 but 040 $b = eng? If this is a lookup we may not be able to do it in phase 1
2. Why are $b and $c not part of the authorized access point when they are for 1XX, 6XX, 7XX agents and works?
3. Deborah's document maps 800, 810, and 811 agents as related agents of the series work, does that need to be added to the mapping as well? 

Thanks!

**Comment by pennylenger** [comment]
Hi Cypress,
1, I think the uri may be http://id.loc.gov.offcampus.lib.washington.edu/authorities/names which means it is constructed according to LCNAF. 
2, All subfields should be included. I just copied this from 830. Sorry! 
3, I can add this.

**Comment by cspayne** [comment]
Thank you Penny! I think we would need to use http://id.loc.gov/authorities/names instead, which doesn't have UW's redirect in the IRI. Should we be implementing this for all agents and works that fit this criteria? In the 1XX, 6XX, 7XX as well? Maybe we can discuss this in our next meeting, it might be a better option over using a string access point instead of a nomen for all of these agents.

**Comment by pennylenger** [comment]
Yes, we can discuss in next meeting, and below are some relevant discussion about this. I hope that it is helpful.

Adam: All of them would be AAPs if in a 100 or 700.  In records where 040 $b = eng, the presumption is that the names are in the LC/NACO NAF or are formulated according to the rules in the 008 or 040 $e (generally, AACR2 or RDA).  For a German library, the assumption is also that they are AAPs but would be found in the German authority file.

I think the assumption is that what is in 1XX 0 or 7XX 0 is an authorized access point, even if there's no authority authorizing it. The 1XX and 7XX are supposed to be for controlled access points, not for any kind of access point. A variant access point should not be put in a 1XX or 7XX.

Gordon: Use AAP for the fields that Adam identified as requiring an LC heading (LCNAF/LCSH) and the source is an LC source.

**Comment by AdamSchiff** [comment]
If there is a series authority record, the IRI should be for that.  For example:

NAR: 100 1#  Alonge, L. J. ǂt Blacktop
URI:  http://id.loc.gov/authorities/names/no2016073110
Used in bib record: 800 1#  Alonge, L. J. ǂt Blacktop ; ǂv 1.

If there isn't a series authority record, I guess we mint a nomen for the series, combining both the author and the title.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by cspayne** [comment]
@AdamSchiff 
Unless an IRI is included in the MARC field, we cannot use it; looking up the authority record is not in scope for phase 1. We can, however, do what Penny has put and provide the source of nomen for 1XX, 7XX, and 8XX where 040 $b = eng as 'http://id.loc.gov/authorities/names' and have these nomens as authorized access points.

**Comment by AdamSchiff** [comment]
The problem with this is that not all name or title access points used in bib records have authority records, so we will be stating that the source is the LC/NACO Authority File when in many cases it isn't.  Catalogers aren't required to do authority records for a name, they're only required to follow the descriptive cataloging standard used (mostly AACR2 or RDA for cataloging done since 1980, and for many earlier descriptions that have been upgraded to one of those standards).  So "100 0" or "100 1" doesn't mean the source of the nomen is LC Names.  It may be and it may not be.  The source might just be the cataloger who followed the rules for formulating a name access point.   And then there are gazillions of bib records from vendors and libraries that don't follow the rules and create crap access points that either aren't formed correctly and/or don't match the authorized form already established in the LC Names file.

This is a conundrum.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***

________________________________

**Comment by cspayne** [comment]
I think this is why we previously decided that if there is no source in $2 then we cannot say that the name or title is an authorized access point and instead of minting a nomen we use an access point datatype property with a string value. This is what is currently implemented in the 1XX and 7XX fields.

**Comment by pennylenger** [comment]
@AdamSchiff @GordonDunsire If it is unsafe and we decide that, even if for headings supposed to be constructed according to LCNAF (no $2 and 040 $e is eng), we do not use AAP for these headings, I can update  these mappings by using an AP datatype property with a string value.

**Comment by AdamSchiff** [comment]
I'm somewhat confused.  I think what is in 800 is intended to be an AAP for the series.  It may have an authority record for the series, or it may not, but it's supposed to be the AAP for the series.  If there's no authority, it's still supposed to be formulated according to RDA instructions for work (series in this case) authorized access points.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by cspayne** [comment]
Based on our previous decision, which we confirmed in yesterday's meeting (2024-10-30), we will not check that 040 $e is eng. We will use an authorized access point when a $0 (approved), $1, or $2 is present, and otherwise will use an access point. 
If a $0 or $1 IRI is used as the agent IRI, we will not mint a nomen for the authorized access point because the source is already provided by the IRI. We will instead use a datatype property for authorized access point and a string value.

@pennylenger can you confirm that this sounds correct?

**Comment by pennylenger** [comment]
Yes, we will not check  040 $e eng for LCNAF. If there is no source, the heading is attached as datatype AP.
But I thought the source was from $2. I am confused by the source in $0 and $1. For example: (previously)
100 1# $a Riera, Jose Joaquin, $d 1969- $1 https://orcid.org/0000-0002-8405-8386

<https://orcid.org/0000-0002-8405-8386> rdaad:P50411 <transform/nomen/blah> . // has authorized access point for person
<transform/nomen/blah> rdand:P80068 "Riera, Jose Joaquin, 1969-" . // has nomen string
<transform/nomen/blah> rdano:P80069 <IRI for LCNAF . // has scheme of nomen

Now, it should be AP and no nomen if there is no $2, but it has $1. But from $1, we know the source of the IRI is ORCID, not that the heading is constructed according to LCNAF. So the source we can get from $1 is not the source of the heading.

**Comment by cspayne** [comment]
hmm I think you are right. But if we are able to get $0's and $1's from id.loc.gov in the input before running the transform, then we could confirm that it is an authorized access point from LCNAF. So maybe we only use aap for $2, or for $0 or $1 that is from id.loc.gov, while other $0 and $1 use a datatype ap?

@CECSpecialistI @GordonDunsire @AdamSchiff @tmqdeborah do you have any thoughts on this?

**Comment by GordonDunsire** [comment]
Transfoming 'source' data: The scope note for Nomen: has scheme of nomen is 'The scheme can be a value or vocabulary encoding scheme or a string encoding scheme.' This allows LCNAF to be recorded as the scheme if the nomen string is constructed with the same rules as entries which have been included in the scheme. That is, if we assume which scheme is being used, or the scheme is recorded explicitly, we do not have to check to see if the nomen string is in the scheme.

In general, if we are uncertain about the 'authorization' of the value of a nomen/nomen string, we should use AP rather than AAP when transforming headings.

No source is specified in the ORCID example, but if subfield $1 has an approved dataset base domain (i.e. 'https://orcid.org/') then the source is known.

We assume that the source indicates the scheme used to construct the nomen string, so we don't mint a nomen to record a 'likely' scheme. It is sufficient to record the heading as a string (datatype) value of an AP/AAP. I don't think we know if ORCID entries are consolidated, in the sense that it is likely that the heading is authorized in ORCID, so we should use AP, not AAP.

**Comment by dchen077** [comment]
@cspayne Hi Cypress, I updated and tested the code for 800, 810, 811. I would love to have you take a look to see if there is anything that needs to be adjusted. Thanks!

**Comment by cspayne** [comment]
Hi @lake44me , I noticed that in the [reproductions guidance wiki](https://github.com/uwlib-cams/MARC2RDA/wiki/Reproductions-Guidance#transformation-guidelines), the transformation guidelines seem to indicate the 8xx fields will have reproduction conditions...do those still need to be added to the mappings?

**Comment by dchen077** [comment]
If I remember correctly from a couple months ago, for 8xx (at least for 800, 810, 811, and 830), there are no mappings that mapped to manifestation and are all mapped to work. That's why they don't have reproduction conditions.

**Comment by dchen077** [comment]
> Hi [@lake44me](https://github.com/lake44me) , I noticed that in the [reproductions guidance wiki](https://github.com/uwlib-cams/MARC2RDA/wiki/Reproductions-Guidance#transformation-guidelines), the transformation guidelines seem to indicate the 8xx fields will have reproduction conditions...do those still need to be added to the mappings?

If I remember correctly from a couple months ago, for 8xx (at least for 800, 810, 811, and 830), there are no mappings that mapped to manifestation and are all mapped to work. That's why they don't have reproduction conditions.

**Comment by cspayne** [comment]
That makes sense! Thanks Doreen.
